Connectivity/Classes/Combine/ConnectivitySubscription.swift

```diff
@@ -17,6 +17,9 @@ class ConnectivitySubscription<S: Subscriber>: Subscription where S.Input == Con
 
     init(configuration: ConnectivityConfiguration, subscriber: S) {
         connectivity = Connectivity(configuration: configuration)
+        connectivity.framework = configuration.framework
+        connectivity.bearerToken = configuration.bearerToken
+        connectivity.authorizationHeader = configuration.authorizationHeader
```

Comment:
There's actually a function in Connectivity.swift called `configure(with:)` which takes the values from the configuration object and applies them - it would sense if these values were applied in the same place.